src/ctr_span.c

```diff
@@ -275,6 +275,11 @@ void ctr_span_set_dropped_events_count(struct ctrace_span *span, uint32_t count)
     span->dropped_events_count = count;
 }
 
+void ctr_span_set_dropped_attributes_count(struct ctrace_span *span, int n)
```

Comment:
all dropped attribute counters are `uint32_t` data type, please adjust the type for `n`

Reply:
Done ✅